content/docs/optimizing-performance.md

```diff
@@ -156,6 +156,19 @@ You can learn more about this in [webpack documentation](https://webpack.js.org/
 
 Remember that you only need to do this for production builds. You shouldn't apply `UglifyJsPlugin` or `DefinePlugin` with `'production'` value in development because they will hide useful React warnings, and make the builds much slower.
 
+### webpack4
+
+In webpack4, instead of using `DefinePlugin`, `'production'` value can be set using `mode` option.  It also provides `UglifyJsPlugin` by default in production mode. 
```

Comment:
Seems like the webpack section was updated and `DefinePlugin` is not present in the documentation now.
https://reactjs.org/docs/optimizing-performance.html#webpack

Thanks for sending PR and sorry for the late turn around time :)